.github/workflows/emscripten.yml

```diff
@@ -32,20 +31,17 @@ jobs:
         cp -r ./out/* ./liblouis-js/js-build
         cp -r ./out-emscripten-install/share/liblouis/tables ./liblouis-js/js-build/
 
-    - name: Debug
-      run: find ./liblouis-js
-
     - name: Echo phantomjs version
       run: phantomjs --version
 
+    # `npm link` exposes the newly build npm package to the test
+    # runner. The `--production`-flag is necessary as it ensures that
+    # no publicly published package of the build is downloaded and
+    # tested instead.
     - name: Build and test the npm package
-      # `npm link` exposes the newly build npm package to the test
-      # runner. The `--production`-flag is necessary as it ensures
-      # that no publicly published package of the build is downloaded
-      # and tested instead.
       run: |
         cd ./liblouis-js
-        # unclear why the npm link is needed
+        # FIXME: unclear why the npm link is needed
```

Comment:
It is explained in the comment above why we use npm link. We don't know whether there is a better solution to link the js-build dependency.